imagepicker/src/main/java/com/esafirm/imagepicker/features/ImagePickerConfig.java

```diff
@@ -8,11 +8,13 @@
 import com.esafirm.imagepicker.features.imageloader.ImageLoader;
 import com.esafirm.imagepicker.model.Image;
 
+import java.io.*;
 import java.util.ArrayList;
 
 public class ImagePickerConfig extends BaseConfig implements Parcelable {
```

Comment:
1. I think we should support `List<String>` as the parameter too. 
2. I looks like `exludedImages` is missing `Parcelable` declaration

Reply:
1. I went with what you used for `selectedImages` which is `ArrayList` only.
2. Do you mean that `File` isn't parcelable? If that's the case we can change it to `String`.

Reply:
1. Fair enough, i haven't had a request to add `List<String>` as the parameter so i think it shouldn't added for now

2. `File` is `Serializeable`  but you have to include it to parcel and extract it too.

ex: 

```java

    @Override
    public void writeToParcel(Parcel dest, int flags) {
        super.writeToParcel(dest, flags);
        dest.writeTypedList(this.selectedImages);
    } 
```

and then to extract it 

```java
protected ImagePickerConfig(Parcel in) {
        this.excludedFiles = new ArrayList<>();
        in.readList(this.excludedFiles, File.class.getClassLoader());
}
```